src/singularity.ts

```diff
@@ -1734,11 +1769,27 @@ const getAvailablePerksDescription = (singularityCount: number): string => {
 function formatPerkDescription(perkData: ISingularityPerkDisplayInfo, singularityCount: number): string {
   const singTolerance = getFastForwardTotalMultiplier()
   const isNew = (singularityCount - perkData.lastUpgraded <= singTolerance)
-  const levelInfo = perkData.currentLevel > 1 ? `<br>${i18next.t('general.level')} ${perkData.currentLevel}` : ''
   //const acquiredUpgraded = ' / Acq ' + perkData.acquired + ' / Upg ' + perkData.lastUpgraded;
-  return `<span${isNew?' class="newPerk"':' class="oldPerk"'} title="${perkData.description}">${perkData.name}${levelInfo}</span>`
+  return `<span${isNew?' class="newPerk"':' class="oldPerk"'} id="${perkData.ID}"> 
+  <img src="Pictures/Default/perk${perkData.ID}.png" loading="lazy">
+  ${perkData.name}
+  </span>`
 }
 
+// Attach EventListener to perks to allow for on hover capabilities.
+export function addPerkEventHandlers() {
+  for (const perk of singularityPerks) {
+    document.getElementById(perk.ID)!.addEventListener('mouseover', () => {
+      const currentLevel = getLastUpgradeInfo(perk, player.highestSingularityCount).level
+      const lastUpgraded = getLastUpgradeInfo(perk, player.highestSingularityCount).singularity
+      const levelInfo = `${i18next.t('general.level')} ${currentLevel} - (Singularity ${lastUpgraded})`
+      const perkImage = DOMCacheGetOrSet('singularityPerksIcon') as HTMLImageElement
+      perkImage.src = `Pictures/Default/perk${perk.ID}.png`
+      DOMCacheGetOrSet('singularityPerksText').textContent = `${levelInfo}`
```

Comment:
```suggestion
      DOMCacheGetOrSet('singularityPerksText').textContent = levelInfo
```

levelInfo is already a string